sourced/ml/transformers/basic.py

```diff
@@ -24,6 +24,19 @@ def __call__(self, head: RDD):
             .csv(self.output)
 
 
+class Mapper(Transformer):
+    """
+    WARNING: Do not use this Transformer for operations more complex than columns renaming
```

Comment:
A-ha! Then let's rename it to `Rower` and it should do `head.map(lambda x: Row(**self.function(x)))`

Reply:
And let's rename `function` to `dicter`

Damn I should spend less time in Go, see what you've done to me @campoy